Aptoide/aptoide-client-v8
# Bugfix for AN-2024: Too many instances of mature switch in home tab

How should this be manually tested?

> Go to the bottom of the home tab and check that mature swith widget exists
> Scroll a little up, enter a random app and press back
> Previously two or more instances of mature switch appear but now only one should be shown

Tickets related to this pull-request: 

- [ ] Fixes #290
